.travis.yml

```diff
@@ -39,7 +39,8 @@ env:
   - OPTIONS="CVR=ON MAKELIB=ON lammps" TEST="_lmp" CHOICES=gfort.lapack
   - LATTE_CMAKE=yes CMAKE_WITH_PROGRESS=yes
   - LATTE_CMAKE=yes LAMMPS_CMAKE=yes
-  - LATTE_PERFORMANCE=yes
+  - LATTE_PERFORMANCE=yes TEST="_lmp" CHOICES=gfort.lapack
+  - LATTE_PERFORMANCE=yes OPTIONS="CVR=ON PROGRESS=ON" TEST="_lmp" CHOICES=gfort.lapack
```

Comment:
How reproducible are the reference performance times on an infrastructure such as Travis-CI? The test on Travis-CI is running on virtualized hardware and we don't control the load on the hypervisor.